### PR TITLE
Mail2kvanitha/c01 aws02

### PR DESCRIPTION
**Pull request by mail2kvanitha**
# Description

Exercise (e.g. C01-E01): c01-aws02

Student Name: Vanitha Kaliyaperumal

> A brief explanation of what has been done.
Create S3 bucket and upload files from EC2 instance to S3 and download files from S3 to EC2 instance.
## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

**Comment by mail2kvanitha** [comment]
projects/project02/application/web/public/favicon.ico file removed from projects

File: classes/01class/exercises/c01-aws02/mail2kvanitha/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+download: s3://mail2kvanitha/test.txt to ./test.txt              
+192-168-1-106:S3ReadOnly vanithak$ ls -l test.txt
+-rw-r--r--  1 vanithak  staff  18 20 Jun 11:49 test.txt
+192-168-1-106:S3ReadOnly vanithak$ 
+
+```
+
+- Add a brief description of the challenges you faced:
+```
+Add comments here
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws02/mail2kvanitha/README.md:168]: Please, let us know about any challenges you had.

File: classes/01class/exercises/c01-aws02/mail2kvanitha/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
+# C01-AWS02
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws02/mail2kvanitha/README.md:1]: The answer is right and well done on that.
Just need some formatting to fix how the file will look like in the repo. Take a look:
https://github.com/devopsacademyau/academy/tree/mail2kvanitha/c01-aws02/classes/01class/exercises/c01-aws02/mail2kvanitha

Comment (mail2kvanitha) [reply classes/01class/exercises/c01-aws02/mail2kvanitha/README.md:1]: Formatting fixed.Please review

**Pull request merged**